src/nidcpower/examples/nidcpower_sink_dc_current_into_electronic_load.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+
+import argparse
+import nidcpower
+import sys
+
+
+def example(
+    resource_name,
+    options,
+    current_level,
+    current_level_range,
+    voltage_limit_range,
+    source_delay,
+    output_shorted,
+    conduction_voltage_mode,
+    conduction_voltage_on_threshold,
+    conduction_voltage_off_threshold,
+    current_level_rising_slew_rate,
+    current_level_falling_slew_rate,
+):
+    with nidcpower.Session(resource_name=resource_name, options=options) as session:
+        # Configure the session.
+        session.source_mode = nidcpower.SourceMode.SINGLE_POINT
+
+        session.output_function = nidcpower.OutputFunction.DC_CURRENT
+        session.current_level = current_level
+        session.current_level_range = current_level_range
+        session.voltage_limit_range = voltage_limit_range
+        # Note that the Voltage Limit property is not applicable for electronic loads and is not
+        # configured in this example. If you change the Output Function, configure the appropriate
+        # level, limit and range properties corresponding to your selected Output Function.
+
+        session.source_delay = source_delay
+
+        # Configure the Output Shorted property to specify whether to simulate a short circuit in
+        # the electronic load.
+        session.output_shorted = output_shorted
+
+        # If you are using the DC Current or Constant Power Output Functions, set the Conduction
+        # Voltage Mode to "Automatic" or "Enabled" to enable Conduction Voltage.
+        # If you are using the DC Voltage or Constant Resistance Output Functions, set the
+        # Conduction Voltage Mode to "Automatic" or "Disabled" to disable Conduction Voltage.
```

Comment:
```suggestion
        # If you are setting the output_function property to nidcpower.OutputFunction.DC_Current
        # or nidcpower.OutputFunction.Constant_Power, set the conduction_voltage_mode
        # to nidcpower.ConductionVoltageMode.Automatic or
        # nidcpower.ConductionVoltageMode.Enabled to enable Conduction Voltage.
        # If you are setting the output_function property to nidcpower.OutputFunction.DC_Voltage
        # or nidcpower.OutputFunction.Constant_Resistance, set the
        # conduction_voltage_mode to nidcpower.ConductionVoltageMode.Automatic or
        # nidcpower.ConductionVoltageMode.Disabled to disable Conduction Voltage.
```

Referring to strings is potentially confusing. The driver will not accept strings. The enum values need to be used.
Similarly, it's not immediately clear that you're referring to settings for the output_function property as opposed to functions/methods in the API.
We may want to take it even further than this and replace